tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/default/unsupported/field_default_model_test.dart

```diff
@@ -37,34 +37,6 @@ void main() {
       },
     );
 
-    test(
```

Comment:
Question: This commit: `test: Moving tests for unsupported default value fields to 'unsupported' folder`/`93c39a78cb64205c95452327d2cb7a701ca01a0f` it is mentioned that some tests are moved, but in the commit they are simply removed. Is this correct?

Reply:
I think this is a mistake from me, this was just me removing tests for the new supported types, instead of including the movement here i just forgot to add it.